packages/data/addon/models/metadata/column.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import { A as arr } from '@ember/array';
+import EmberObject from '@ember/object';
+
+class Column extends EmberObject {
+  /**
+   * @property {String} identifierField - field used as the id
+   */
+  static identifierField = 'id';
+
+  /**
+   * @property {String} id
+   */
+  id;
+
+  /**
+   * @property {String} name - Display name
+   */
+  name;
+
+  /**
+   * @property {String} description
+   */
+  description;
+
+  /**
+   * @property {Table} table
```

Comment:
Wonder if we should add `@typedef` to these? https://jsdoc.app/tags-typedef.html

Reply:
I think in this case it's fine because it's referencing another model in the same directory